web/themes/custom/liibarrio2020/scss/abstracts/_variables.scss

```diff
@@ -1,7 +1,7 @@
 /* Override the Bootstrap 4 variables
 -----------------------------------------*/
 // Google Fonts
-@import url("https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&family=PT+Serif:ital,wght@0,400;0,700;1,400;1,700&display=swap');
```

Comment:
https://github.com/laws-africa/la-web-components/issues/29#issuecomment-965184335
or should we use local fonts?

Reply:
I think this is suitable.